### ISSUE TITLE
Empty value in ids filter leads to 500 error

### ISSUE THREAD

**Issue by ar7n** [barseghyanartur/django-elasticsearch-dsl-drf#130]
When i use IdsFilterBackend and send query with empty ids param, i've got 500 error:

```
RequestError at /api/tags/
RequestError(400, 'search_phase_execution_exception', 'failed to create query: {\n  "ids" : {\n    "type" : [\n      "doc"\n    ],\n    "values" : [\n      ""\n    ],\n    "boost" : 1.0\n  }\n}')

Request Method: GET
Request URL: https://localhost:3000/api/tags/?ids=
Django Version: 2.2
Python Executable: /usr/local/bin/python
Python Version: 3.7.0
Python Path: ['/app', '/app', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/usr/local/lib/python3.7/site-packages']
Server time: Пн, 22 Апр 2019 19:06:58 +0300
Installed Applications:
['modeltranslation',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'django_elasticsearch_dsl',
 'imagekit',
 'rest_framework',
 'rest_framework.authtoken',
 'django_elasticsearch_dsl_drf',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'allauth.socialaccount.providers.vk',
 'allauth.socialaccount.providers.odnoklassniki',
 'allauth.socialaccount.providers.google',
 'allauth.socialaccount.providers.facebook',
 'allauth.socialaccount.providers.twitter',
 'rest_auth',
 'rest_auth.registration',
 'djcelery_email',
 'adminsortable2',
 'apps.common.apps.CommonAppConfig',
 'apps.generator.apps.CoreAppConfig',
 'apps.users.apps.UserAppConfig',
 'apps.tags.apps.TagsAppConfig',
 'apps.products.apps.ProductsAppConfig',
 'debug_toolbar',
 'elastic_panel']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'apps.currencies.middleware.CurrencyMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware']


Traceback:

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python3.7/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/rest_framework/viewsets.py" in view
  116.             return self.dispatch(request, *args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py" in dispatch
  495.             response = self.handle_exception(exc)

File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py" in handle_exception
  455.             self.raise_uncaught_exception(exc)

File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py" in dispatch
  492.             response = handler(request, *args, **kwargs)

File "/usr/local/lib/python3.7/site-packages/rest_framework/mixins.py" in list
  42.         page = self.paginate_queryset(queryset)

File "/usr/local/lib/python3.7/site-packages/rest_framework/generics.py" in paginate_queryset
  173.         return self.paginator.paginate_queryset(queryset, self.request, view=self)

File "/usr/local/lib/python3.7/site-packages/django_elasticsearch_dsl_drf/pagination.py" in paginate_queryset
  157.             self.page = paginator.page(page_number)

File "/usr/local/lib/python3.7/site-packages/django_elasticsearch_dsl_drf/pagination.py" in page
  51.         number = self.validate_number(number)

File "/usr/local/lib/python3.7/site-packages/django/core/paginator.py" in validate_number
  48.         if number > self.num_pages:

File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py" in __get__
  80.         res = instance.__dict__[self.name] = self.func(instance)

File "/usr/local/lib/python3.7/site-packages/django/core/paginator.py" in num_pages
  97.         if self.count == 0 and not self.allow_empty_first_page:

File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py" in __get__
  80.         res = instance.__dict__[self.name] = self.func(instance)

File "/usr/local/lib/python3.7/site-packages/django/core/paginator.py" in count
  91.             return c()

File "/usr/local/lib/python3.7/site-packages/elasticsearch_dsl/search.py" in count
  660.             **self._params

File "/usr/local/lib/python3.7/site-packages/elasticsearch/client/utils.py" in _wrapped
  76.             return func(*args, params=params, **kwargs)

File "/usr/local/lib/python3.7/site-packages/elasticsearch/client/__init__.py" in count
  1112.             doc_type, '_count'), params=params, body=body)

File "/usr/local/lib/python3.7/site-packages/elasticsearch/transport.py" in perform_request
  318.                 status, headers_response, data = connection.perform_request(method, url, params, body, headers=headers, ignore=ignore, timeout=timeout)

File "/usr/local/lib/python3.7/site-packages/elasticsearch/connection/http_urllib3.py" in perform_request
  186.             self._raise_error(response.status, raw_data)

File "/usr/local/lib/python3.7/site-packages/elasticsearch/connection/base.py" in _raise_error
  125.         raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)

Exception Type: RequestError at /api/tags/
Exception Value: RequestError(400, 'search_phase_execution_exception', 'failed to create query: {\n  "ids" : {\n    "type" : [\n      "doc"\n    ],\n    "values" : [\n      ""\n    ],\n    "boost" : 1.0\n  }\n}')
Request information:
USER: admin

GET:
ids = ''

POST: No POST data

FILES: No FILES data

COOKIES:
_ga = 'GA1.1.1297572531.1545419964'
csrftoken = 'nXMpJqjYHCCmg0pJZvyq0IAVQx9Cr91eD4MbeKbGgSFs0K0kVVvsVqyuTmXK2AmN'
sessionid = 'jd1lwenw1p5e1p61enlqdwb6bcwhp38y'
tabstyle = 'html-tab'
currency = 'USD'
djdt = 'hide'

META:
CONTENT_LENGTH = ''
CONTENT_TYPE = 'text/plain'
CSRF_COOKIE = 'nXMpJqjYHCCmg0pJZvyq0IAVQx9Cr91eD4MbeKbGgSFs0K0kVVvsVqyuTmXK2AmN'
DJANGO_SETTINGS_MODULE = 'config.settings'
EMAIL_HOST = 'smtp.mailgun.org'
EMAIL_HOST_PASSWORD = '********************************'
EMAIL_HOST_USER = 'postmaster@mail.idacloud.ru'
GATEWAY_INTERFACE = 'CGI/1.1'
GPG_KEY = '0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D'
HOME = '/root'
HOSTNAME = '0f14618c9c52'
HTTP_ACCEPT = '*/*'
HTTP_ACCEPT_ENCODING = 'gzip, deflate, br'
HTTP_ACCEPT_LANGUAGE = 'ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7'
HTTP_CACHE_CONTROL = 'no-cache'
HTTP_CONNECTION = 'close'
HTTP_COOKIE = '_ga=GA1.1.1297572531.1545419964; csrftoken=nXMpJqjYHCCmg0pJZvyq0IAVQx9Cr91eD4MbeKbGgSFs0K0kVVvsVqyuTmXK2AmN; sessionid=jd1lwenw1p5e1p61enlqdwb6bcwhp38y; tabstyle=html-tab; currency=USD; djdt=hide'
HTTP_HOST = 'localhost'
HTTP_PRAGMA = 'no-cache'
HTTP_REFERER = 'https://localhost:3000/api/tags/?ids='
HTTP_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36'
HTTP_X_FORWARDED_HOST = 'localhost:3000'
HTTP_X_FORWARDED_PORT = '3000'
HTTP_X_FORWARDED_PROTO = 'https'
HTTP_X_REAL_IP = '172.22.0.1'
HTTP_X_REQUESTED_WITH = 'XMLHttpRequest'
IMAGE_FACTORY_URL = 'http://loremflickr.com/{width}/{height}/{theme}'
LANG = 'C.UTF-8'
PATH = '/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
PATH_INFO = '/api/tags/'
PWD = '/app'
PYTHONUNBUFFERED = '1'
PYTHON_PIP_VERSION = '18.1'
PYTHON_VERSION = '3.7.0'
QUERY_STRING = 'ids='
REMOTE_ADDR = '172.22.0.9'
REMOTE_HOST = ''
REQUEST_METHOD = 'GET'
RUN_MAIN = 'true'
SCRIPT_NAME = ''
SERVER_NAME = '0f14618c9c52'
SERVER_PORT = '8000'
SERVER_PROTOCOL = 'HTTP/1.0'
SERVER_SOFTWARE = 'WSGIServer/0.2'
SHLVL = '2'
TZ = 'Europe/Moscow'
wsgi.errors = <_io.TextIOWrapper name='<stderr>' mode='w' encoding='UTF-8'>
wsgi.file_wrapper = ''
wsgi.input = <django.core.handlers.wsgi.LimitedStream object at 0x7f4e0fe2b240>
wsgi.multiprocess = False
wsgi.multithread = True
wsgi.run_once = False
wsgi.url_scheme = 'http'
wsgi.version = '(1, 0)'

Settings:
Using settings module config.settings
ABSOLUTE_URL_OVERRIDES = {}
ACCOUNT_AUTHENTICATION_METHOD = 'username_email'
ACCOUNT_EMAIL_REQUIRED = True
ADMINS = []
ALLOWED_HOSTS = ['*']
APPEND_SLASH = True
AUTHENTICATION_BACKENDS = "('django.contrib.auth.backends.ModelBackend', 'apps.users.auth_backends.CustomAuthenticationBackend')"
AUTH_PASSWORD_VALIDATORS = '********************'
AUTH_USER_MODEL = 'users.User'
BASE_DIR = '/app'
CACHES = {'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}}
CACHE_MIDDLEWARE_ALIAS = 'default'
CACHE_MIDDLEWARE_KEY_PREFIX = '********************'
CACHE_MIDDLEWARE_SECONDS = 600
CELERY_ACCEPT_CONTENT = ['json']
CELERY_BROKER_URL = 'redis://redis:6379/0'
CELERY_RESULT_BACKEND = 'redis://redis:6379/0'
CELERY_RESULT_SERIALIZER = 'json'
CELERY_TASK_SERIALIZER = 'json'
CELERY_TIMEZONE = 'Europe/Moscow'
CSRF_COOKIE_AGE = 31449600
CSRF_COOKIE_DOMAIN = None
CSRF_COOKIE_HTTPONLY = False
CSRF_COOKIE_NAME = 'csrftoken'
CSRF_COOKIE_PATH = '/'
CSRF_COOKIE_SAMESITE = 'Lax'
CSRF_COOKIE_SECURE = False
CSRF_FAILURE_VIEW = 'django.views.csrf.csrf_failure'
CSRF_HEADER_NAME = 'HTTP_X_CSRFTOKEN'
CSRF_TRUSTED_ORIGINS = []
CSRF_USE_SESSIONS = False
CURRENCIES = "(('RUB', 'Russian Ruble'), ('USD', 'US Dollar'), ('EUR', 'Euro'))"
CURRENCY_CODE = 'RUB'
CURRENCY_COOKIE_NAME = 'currency'
DATABASES = {'default': {'ENGINE': 'django.db.backends.postgresql', 'NAME': 'postgres', 'USER': 'postgres', 'HOST': 'db', 'PORT': 5432, 'CONN_MAX_AGE': 600, 'ATOMIC_REQUESTS': False, 'AUTOCOMMIT': True, 'OPTIONS': {}, 'TIME_ZONE': None, 'PASSWORD': '********************', 'TEST': {'CHARSET': None, 'COLLATION': None, 'NAME': None, 'MIRROR': None}}}
DATABASE_ROUTERS = []
DATA_UPLOAD_MAX_MEMORY_SIZE = 2621440
DATA_UPLOAD_MAX_NUMBER_FIELDS = 1000
DATETIME_FORMAT = 'N j, Y, P'
DATETIME_INPUT_FORMATS = ['%Y-%m-%d %H:%M:%S', '%Y-%m-%d %H:%M:%S.%f', '%Y-%m-%d %H:%M', '%Y-%m-%d', '%m/%d/%Y %H:%M:%S', '%m/%d/%Y %H:%M:%S.%f', '%m/%d/%Y %H:%M', '%m/%d/%Y', '%m/%d/%y %H:%M:%S', '%m/%d/%y %H:%M:%S.%f', '%m/%d/%y %H:%M', '%m/%d/%y']
DATE_FORMAT = 'N j, Y'
DATE_INPUT_FORMATS = ['%Y-%m-%d', '%m/%d/%Y', '%m/%d/%y', '%b %d %Y', '%b %d, %Y', '%d %b %Y', '%d %b, %Y', '%B %d %Y', '%B %d, %Y', '%d %B %Y', '%d %B, %Y']
DEBUG = True
DEBUG_PROPAGATE_EXCEPTIONS = False
DEBUG_TOOLBAR_CONFIG = {'SHOW_TOOLBAR_CALLBACK': 'config.settings.show_toolbar_callback'}
DEBUG_TOOLBAR_PANELS = ['debug_toolbar.panels.versions.VersionsPanel', 'debug_toolbar.panels.timer.TimerPanel', 'debug_toolbar.panels.settings.SettingsPanel', 'debug_toolbar.panels.headers.HeadersPanel', 'debug_toolbar.panels.request.RequestPanel', 'debug_toolbar.panels.sql.SQLPanel', 'debug_toolbar.panels.staticfiles.StaticFilesPanel', 'debug_toolbar.panels.templates.TemplatesPanel', 'debug_toolbar.panels.cache.CachePanel', 'debug_toolbar.panels.signals.SignalsPanel', 'debug_toolbar.panels.logging.LoggingPanel', 'debug_toolbar.panels.redirects.RedirectsPanel', 'elastic_panel.panel.ElasticDebugPanel']
DECIMAL_SEPARATOR = '.'
DEFAULT_CHARSET = 'utf-8'
DEFAULT_CONTENT_TYPE = 'text/html'
DEFAULT_EXCEPTION_REPORTER_FILTER = 'django.views.debug.SafeExceptionReporterFilter'
DEFAULT_FILE_STORAGE = 'django_hashedfilenamestorage.storage.HashedFilenameFileSystemStorage'
DEFAULT_FROM_EMAIL = 'webmaster@localhost'
DEFAULT_INDEX_TABLESPACE = ''
DEFAULT_TABLESPACE = ''
DISALLOWED_USER_AGENTS = []
ELASTICSEARCH_DSL = {'default': {'hosts': 'elastic:9200'}}
EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
EMAIL_HOST = 'smtp.mailgun.org'
EMAIL_HOST_PASSWORD = '********************'
EMAIL_HOST_USER = 'postmaster@mail.idacloud.ru'
EMAIL_PORT = 25
EMAIL_SSL_CERTFILE = None
EMAIL_SSL_KEYFILE = '********************'
EMAIL_SUBJECT_PREFIX = 'handma.de: '
EMAIL_TIMEOUT = None
EMAIL_USE_LOCALTIME = False
EMAIL_USE_SSL = False
EMAIL_USE_TLS = True
FILE_CHARSET = 'utf-8'
FILE_UPLOAD_DIRECTORY_PERMISSIONS = 493
FILE_UPLOAD_HANDLERS = ['django.core.files.uploadhandler.MemoryFileUploadHandler', 'django.core.files.uploadhandler.TemporaryFileUploadHandler']
FILE_UPLOAD_MAX_MEMORY_SIZE = 2621440
FILE_UPLOAD_PERMISSIONS = 420
FILE_UPLOAD_TEMP_DIR = None
FIRST_DAY_OF_WEEK = 0
FIXTURE_DIRS = []
FORCE_SCRIPT_NAME = None
FORMAT_MODULE_PATH = None
FORM_RENDERER = 'django.forms.renderers.DjangoTemplates'
IGNORABLE_404_URLS = []
IMAGEKIT_CACHEFILE_DIR = 'c'
IMAGEKIT_CACHEFILE_NAMER = 'imagekit.cachefiles.namers.hash'
IMAGEKIT_CACHE_BACKEND = 'default'
IMAGEKIT_CACHE_PREFIX = 'imagekit:'
IMAGEKIT_CACHE_TIMEOUT = 300
IMAGEKIT_DEFAULT_CACHEFILE_BACKEND = 'imagekit.cachefiles.backends.Simple'
IMAGEKIT_DEFAULT_CACHEFILE_STRATEGY = 'imagekit.cachefiles.strategies.Optimistic'
IMAGEKIT_DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
IMAGEKIT_SPEC_CACHEFILE_NAMER = 'apps.common.namers.namer'
IMAGEKIT_USE_MEMCACHED_SAFE_CACHE_KEY = '********************'
IMAGE_FACTORY_URL = 'http://loremflickr.com/{width}/{height}/{theme}'
INSTALLED_APPS = ['modeltranslation', 'django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'django.contrib.sites', 'django_elasticsearch_dsl', 'imagekit', 'rest_framework', 'rest_framework.authtoken', 'django_elasticsearch_dsl_drf', 'allauth', 'allauth.account', 'allauth.socialaccount', 'allauth.socialaccount.providers.vk', 'allauth.socialaccount.providers.odnoklassniki', 'allauth.socialaccount.providers.google', 'allauth.socialaccount.providers.facebook', 'allauth.socialaccount.providers.twitter', 'rest_auth', 'rest_auth.registration', 'djcelery_email', 'adminsortable2', 'apps.common.apps.CommonAppConfig', 'apps.generator.apps.CoreAppConfig', 'apps.users.apps.UserAppConfig', 'apps.tags.apps.TagsAppConfig', 'apps.products.apps.ProductsAppConfig', 'debug_toolbar', 'elastic_panel']
INTERNAL_IPS = []
LANGUAGES = "(('ru', 'Русский'), ('en', 'Английский'))"
LANGUAGES_BIDI = ['he', 'ar', 'fa', 'ur']
LANGUAGE_CODE = 'ru'
LANGUAGE_COOKIE_AGE = None
LANGUAGE_COOKIE_DOMAIN = None
LANGUAGE_COOKIE_NAME = 'lang'
LANGUAGE_COOKIE_PATH = '/'
LOCALE_PATHS = "('./locale',)"
LOGGING = {}
LOGGING_CONFIG = 'logging.config.dictConfig'
LOGIN_REDIRECT_URL = '/accounts/profile/'
LOGIN_URL = '/accounts/login/'
LOGOUT_REDIRECT_URL = None
MANAGERS = []
MEDIA_ROOT = '/app/media'
MEDIA_URL = '/m/'
MESSAGE_STORAGE = 'django.contrib.messages.storage.fallback.FallbackStorage'
MIDDLEWARE = ['django.middleware.security.SecurityMiddleware', 'django.contrib.sessions.middleware.SessionMiddleware', 'django.middleware.locale.LocaleMiddleware', 'django.middleware.common.CommonMiddleware', 'django.middleware.csrf.CsrfViewMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware', 'django.middleware.clickjacking.XFrameOptionsMiddleware', 'apps.currencies.middleware.CurrencyMiddleware', 'debug_toolbar.middleware.DebugToolbarMiddleware']
MIGRATION_MODULES = {}
MONTH_DAY_FORMAT = 'F j'
NUMBER_GROUPING = 0
PASSWORD_HASHERS = '********************'
PASSWORD_RESET_TIMEOUT_DAYS = '********************'
PREPEND_WWW = False
RATES_URL = 'http://www.cbr.ru/scripts/XML_daily.asp'
REST_AUTH_SERIALIZERS = {'LOGIN_SERIALIZER': 'apps.users.serializers.RememberLoginSerializer', 'USER_DETAILS_SERIALIZER': 'apps.users.serializers.UserDetailsSerializer'}
REST_FRAMEWORK = {'PAGE_SIZE': 12, 'DEFAULT_AUTHENTICATION_CLASSES': ('rest_framework.authentication.SessionAuthentication',), 'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination'}
ROOT_URLCONF = 'config.urls'
SECRET_KEY = '********************'
SECURE_BROWSER_XSS_FILTER = False
SECURE_CONTENT_TYPE_NOSNIFF = False
SECURE_HSTS_INCLUDE_SUBDOMAINS = False
SECURE_HSTS_PRELOAD = False
SECURE_HSTS_SECONDS = 0
SECURE_PROXY_SSL_HEADER = "('HTTP_X_FORWARDED_PROTO', 'https')"
SECURE_REDIRECT_EXEMPT = []
SECURE_SSL_HOST = None
SECURE_SSL_REDIRECT = False
SERVER_EMAIL = 'root@localhost'
SESSION_CACHE_ALIAS = 'default'
SESSION_COOKIE_AGE = 31536000
SESSION_COOKIE_DOMAIN = None
SESSION_COOKIE_HTTPONLY = True
SESSION_COOKIE_NAME = 'sessionid'
SESSION_COOKIE_PATH = '/'
SESSION_COOKIE_SAMESITE = 'Lax'
SESSION_COOKIE_SECURE = False
SESSION_ENGINE = 'django.contrib.sessions.backends.db'
SESSION_EXPIRE_AT_BROWSER_CLOSE = False
SESSION_FILE_PATH = None
SESSION_SAVE_EVERY_REQUEST = False
SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
SETTINGS_MODULE = 'config.settings'
SHORT_DATETIME_FORMAT = 'm/d/Y P'
SHORT_DATE_FORMAT = 'm/d/Y'
SIGNING_BACKEND = 'django.core.signing.TimestampSigner'
SILENCED_SYSTEM_CHECKS = "('rest_framework.W001',)"
SITE_ID = 1
SOCIALACCOUNT_ADAPTER = 'apps.users.adapters.SocialAccountAdapter'
STATICFILES_DIRS = []
STATICFILES_FINDERS = ['django.contrib.staticfiles.finders.FileSystemFinder', 'django.contrib.staticfiles.finders.AppDirectoriesFinder']
STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
STATIC_ROOT = '/app/static'
STATIC_URL = '/s/'
TEMPLATES = [{'BACKEND': 'django.template.backends.django.DjangoTemplates', 'DIRS': ['/app/templates'], 'APP_DIRS': True, 'OPTIONS': {'context_processors': ['django.template.context_processors.debug', 'django.template.context_processors.request', 'django.contrib.auth.context_processors.auth', 'django.contrib.messages.context_processors.messages']}}]
TEST_NON_SERIALIZED_APPS = []
TEST_RUNNER = 'django.test.runner.DiscoverRunner'
THOUSAND_SEPARATOR = ','
TIME_FORMAT = 'P'
TIME_INPUT_FORMATS = ['%H:%M:%S', '%H:%M:%S.%f', '%H:%M']
TIME_ZONE = 'Europe/Moscow'
USE_I18N = True
USE_L10N = True
USE_THOUSAND_SEPARATOR = False
USE_TZ = True
USE_X_FORWARDED_HOST = True
USE_X_FORWARDED_PORT = True
WSGI_APPLICATION = 'config.wsgi.application'
X_FRAME_OPTIONS = 'SAMEORIGIN'
YEAR_MONTH_FORMAT = 'F Y'
```

I can help with this bug

**Comment by barseghyanartur** [comment]
@ar7n:

First of all, thanks for your efforts!

Regarding the issue. I don't get such errors with empty `ids` given.

In the example project, the following link does not fail:

- http://localhost:8000/search/books/?ids=

There are also checks for empty `ids`:

- https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/filter_backends/filtering/ids.py#L109

Thus, empty values shall not cause issues. Could you come up with a test case for this?

**Comment by barseghyanartur** [comment]
@ar7n:

P. S.

If you have some free time to spend on this project (for fun or just because you like to contribute), please contact me (see the AUTHOR section of the README.rst).

**Comment by barseghyanartur** [comment]
Any progress on this? As I mentioned already, I can't reproduce it.

**Comment by ar7n** [comment]
@barseghyanartur it seems like this bug occurs only with elasticsearch 6 and elasticsearch-dsl-py 6. I've switched my version to 5 and everything stared to work well.

**Comment by barseghyanartur** [comment]
@ar7n:

That's very weird anyway, since it's not supposed to happen. Nowadays I'm using 6.x branch preliminary, keeping older versions for tests only (maintenance mode).

**Comment by barseghyanartur** [comment]
@ar7n:

Reproduced it on Elasticsearch 6.4.x. 

You're more than welcome to make a pull request. When (if) you do, please, add a test for this somewhere here [django_elasticsearch_dsl_drf.tests.test_filtering_common.TestFilteringCommon#test_ids_filter](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/tests/test_filtering_common.py#L523) (as `test_ids_filter_no_value`).

**Comment by barseghyanartur** [comment]
@ar7n:

Fixed in 0.17.7.
